src/components/DataTables.vue

```diff
@@ -104,14 +104,15 @@
             return props.some(prop => {
               let elVal = el[prop]
               /* istanbul ignore if */
-              if (elVal === undefined) {
-                console.error(ErrorTips.propError(prop))
```

Comment:
it seems I miss a `return false`. can you help to add? Please keep this console.error, I want provide a special reminder to the user.

Reply:
Fixed. But in this case each type can produce thousands warnings in console, for example: ![Image](http://dl3.joxi.net/drive/2017/11/13/0003/0636/258684/84/58aafae0e1.jpg)

Reply:
What do you think about such solution?

```
diff --git a/src/components/DataTables.vue b/src/components/DataTables.vue
index fac7c21..66f39ec 100644
--- a/src/components/DataTables.vue
+++ b/src/components/DataTables.vue
@@ -93,6 +93,8 @@
           })
         }
 
+        let propErrors = {}
+
         this.filters.forEach(filter => {
           let vals = filter.vals
           if (!vals || vals.length === 0) {
@@ -105,7 +107,10 @@
               let elVal = el[prop]
               /* istanbul ignore if */
               if (elVal === undefined) {
-                console.error(ErrorTips.propError(prop))
+                if (!propErrors[prop]) {
+                  propErrors[prop] = true
+                  console.error(ErrorTips.propError(prop))
+                }
                 return false
               } else if (elVal === null) {
                 return false
```
it will fire warning once per prop